openfisca_aotearoa/variables/acts/super/super.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+
+# Import from openfisca-core the common python objects used to code the legislation in OpenFisca
+from openfisca_core.model_api import *
+# Import the entities specifically defined for this tax and entitlement system
+from openfisca_aotearoa.entities import Person
+
+
+class super__living_alone(Variable):
+    value_type = int
+    entity = Person
+    definition_period = MONTH
+    label = u"Is a person considered as living alone"
+    reference = "http://www.legislation.govt.nz/act/public/2001/0084/latest/whole.html#DLM5578822"
+
+
+class super__is_receiving_compensation(Variable):
```

Comment:
I wonder if this variable should be called acc__is_receving_compensation in that the reference link is to the ACC act?